subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -94,7 +93,6 @@ public abstract class DefaultGradle extends AbstractPluginAware implements Gradl
     private final MutableActionSet<Project> rootProjectActions = new MutableActionSet<>();
     private final IsolatedProjectEvaluationListenerProvider isolatedProjectEvaluationListenerProvider;
     private GradleLifecycle lifecycle;
-    private boolean projectsLoaded;
```

Comment:
Don't need any extra state. The state we're looking for is already bear by `baseProjectClassLoaderScope`